ksql-engine/src/test/java/io/confluent/ksql/TopologyFileGenerator.java

```diff
@@ -49,15 +52,25 @@
  */
 public final class TopologyFileGenerator {
 
-    private static final String BASE_DIRECTORY = "ksql-engine/src/test/resources/expected_topology/";
+    private static final String BASE_DIRECTORY = "src/test/resources/expected_topology/";
 
-    private TopologyFileGenerator() {
+    @Ignore // comment me out to generate the persisted topologies
+    @Test
+    public void doGenerateTopologies() throws Exception {
```

Comment:
Rather than having a test that needs to be un-ignored, why not just leave a `main` method? It's nice to be able to generate the topologies without having to change the code... especially if we need to script it.  It's only a matter of time before this would get checked in with the ignore commented out ;)

Reply:
The problem is that using `main()` doesn't use the same current working directory that junit uses. With the `main` cwd, we can't discover the `udf-example.jar` that has our custom UDFs. Thoughts on a good way to fix this?